crates/shimkit/Cargo.toml

```diff
@@ -0,0 +1,109 @@
+[package]
```

Comment:
Would like to have a CHANGELOG.md file in this crate and a would like to discuss how this will change our release process defined in the release.md.

I'd propose we still do monthly release with the two crates releasing at the same time. What do you think?